client/components/ScenariosList/index.jsx

```diff
@@ -13,9 +19,19 @@ const ScenarioEntries = ({ scenarioData, isLoggedIn }) => {
 
     return scenarioData.map(({ id, title, description }) => {
         return (
-            <Grid.Column width={4} key={id}>
+            <Grid.Column
+                mobile={16}
+                tablet={8}
+                computer={5}
+                key={id}
+                className="tm__scenario-entry"
+            >
                 <Header as="h3">{title}</Header>
-                <p>{description}</p>
+                <p className="tm__scenario-desc">
```

Comment:
I'd rather see this logic be split into a component, I'm not a huge fan of seeing ternary expressions inside `{}` inside JSX, it gets really difficult to read.

```js
const Description = ({ description, desiredLength, as: Component, ...rest }) => {
  let display = description;
  if (display.length > desiredLength) {
    display = `${display.substr(0, desiredLength)}...`;
  }
  return <Component {...rest}>{display}</Component>;
}

Description.defaultProps = {
  as: "p"
};
```

Then you can just replace this whole chunk with `<Description className="tm__scenario-desc" description={description} desiredLength={210} />` or whatever...


Also - instead of specififying a number of characters and doing string substring stuff, there are css tricks you could do to "fade out" the last line of text https://css-tricks.com/text-fade-read-more/

Reply:
@gnarf I just used overflow-y. See updated demo